packages/cli/src/bin.ts

```diff
@@ -27,7 +27,7 @@ import { DebugInfo } from './DebugInfo'
 import { Format } from './Format'
 import { Generate } from './Generate'
 import { Init } from './Init'
-import { Platform } from './platform'
+import { Platform } from './platform/*Platform'
```

Comment:
This is what is breaking all the tests I guess

Reply:
Surprised. I am not aware of file name rules. Trying different things now. Whatever tools are being broken by this are the tools' fault AFAICT. Anyways we don't need to enter this rabbit hole. If `*` is the issue it shall be removed at present.

Reply:
Not the best time to fix the tooling indeed

Reply:
Working with `_` :)